README.md

```diff
@@ -37,14 +53,15 @@ At that point, you will be able to run the django server normally (within the pi
 And your server should be available at http://localhost:8000.
 
 ## Starting up a droplet
+
 This deployment has been tested using DigitalOcean with Docker.
 
 To start up the server from scratch on a droplet:
 
 ```
-git pull https://github.com/eepMoody/open5e-api
+git pull https://github.com/open5e/open5e-api
 export SECRET_KEY=a_new_secret_key
 export SERVER_NAME=whatever.open5e.com
```

Comment:
```suggestion
export SERVER_NAME=whatever.yourdomain.com
```

A small change (since most consumers won't be using open5e.com) that has the added benefit of kicking off a new CI/CD build! Sorry about that disruption, switching to an org had some unintended side effects.